applications/ParticleMechanicsApplication/custom_conditions/particle_based_conditions/mpm_particle_penalty_coupling_interface_condition.cpp

```diff
@@ -212,13 +211,11 @@ void MPMParticlePenaltyCouplingInterfaceCondition::CalculateInterfaceContactForc
     if (Is(CONTACT))
     {
         // Apply only in the normal direction
-        array_1d<double, 3 > & unit_normal_vector = this->GetValue(MPC_NORMAL);
-        ParticleMechanicsMathUtilities<double>::Normalize(unit_normal_vector);
-        const double normal_force = MathUtils<double>::Dot(mpc_force,unit_normal_vector);
+        const double normal_force = MathUtils<double>::Dot(mpc_force, m_unit_normal);
```

Comment:
is m_unit_normal normalized? Can you always Normalize to make sure? This is essentially needed if we do a coupling with FEM, cause FEM will only send the direction.

Reply:
Yep its normalized on input.
https://github.com/KratosMultiphysics/Kratos/blob/2cc6f47d6bbc432f007d7816e3e77642279c31f7/applications/ParticleMechanicsApplication/custom_conditions/particle_based_conditions/mpm_particle_penalty_dirichlet_condition.cpp#L314